dojo/templates/dojo/view_group.html

```diff
@@ -118,7 +118,7 @@ <h4 class="pull-left">Members of this Group</h4>
                 <div class="clearfix">
                     <h4 class="pull-left">Product Types this Group can access</h4>
                     &nbsp;
-                    <a href="https://documentation.defectdojo.com/usage/permissions/" target="_blank">
+                    <a href="https://docs.defectdojo.com/en/customize_dojo/user_management/create_user_group/#add-product-roles-or-product-type-roles-for-a-group" target="_blank">
```

Comment:
Should the two links that go to the same location be combined? I initially found it confusing that I was in the same place until I realized it was under one umbrella.

"Product Types and Products this Group can access"

Reply:
No, they should not. They are two separate places in the UI to list different types of group relationships. One is listing Product Type Groups, and the other is listing Product Groups. The link does take you to the same location in the documentation for both sections, but that is because the documentation describes Product Roles and Product Type Roles for Groups in one place.

<img width="763" alt="Screenshot 2025-03-13 at 11 39 13 AM" src="https://github.com/user-attachments/assets/9cd8ac39-6cd0-4bb7-95ad-215d5615b852" />